css-layout-classes/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>Layout</title>
+  </head>
+
+  <body>
+
+    <div id="body">
```

Comment:
I would change this to your `.container` class. You should have 1 container div below the body that holds all of the content